recipes/fipy/LICENSE.rst

```diff
@@ -0,0 +1,32 @@
+------------
```

Comment:
Is this file in the source tarball? If so, you don't need this in the recipe

Reply:
It is in the tarball. I've removed `LICENSE.rst`.